### PR TITLE
Add notification jobs to github actions

### PR DESCRIPTION
**Pull request by tlangs**
We want `datarepo-helm` to alert us when _things_ happen.

### CODE REVIEW

File: .github/workflows/chart-releaser.yaml [hunk 1]
```diff
@@ -5,7 +5,10 @@ on:
 
 jobs:
   release_chart:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```
Comment (ddietterich) [inline .github/workflows/chart-releaser.yaml:11]: This seems like clutter to me. Until there is more than one `runs-on`, it is just noise.

**Review by ddietterich — COMMENTED** [review]
Hey, I made a comment :)

File: .github/workflows/chart-releaser.yaml (hunk 1 above)
Comment (tlangs) [reply .github/workflows/chart-releaser.yaml:11]: https://github.com/broadinstitute/datarepo-helm/pull/167/files#diff-846c783941a119f756e5ec0848910126201b207caeb37ffd52c139b34a23f98aR44 is copied from our other github actions, and the comment says `matrix` needs to be there. I don't know a lot about actions, so I just added it. If we don't actually need `matrix`, that's great and I can remove it!

Comment (ddietterich) [reply .github/workflows/chart-releaser.yaml:11]: `matrix` is the way GHA give you to run the same script with different arguments.

We use it in WSM to run a test script and have it run `unit` and `connected`. If you wanted to test on more than one OS, you could use matrix to say, "run this workflow on ubuntu and run it on Windows 11". When you only have one option, `matrix` doesn't make sense.

Here is an example slack notification from a WSM workflow that doesn't mention matrix at all: [nightly-tests](https://github.com/DataBiosphere/terra-workspace-manager/blob/0f13be7f13b3e4535930022d7edf9cfd38a8509e/.github/workflows/nightly-tests.yml#L105)

Comment (ddietterich) [reply .github/workflows/chart-releaser.yaml:11]: Sadly, I have been spending way too much time on GHA. They are pretty hard to debug. Happy to chat if I can be of help.

Comment (snf2ye) [reply .github/workflows/chart-releaser.yaml]: I think I added that original comment - I vaguely remember finding a source that suggested referencing the matrix would get the "job" field to show up in the slack messages.  But, maybe that's only for jobs that are already using the matrix variable? Definitely worth testing to see if you need it.

**Review by snf2ye — APPROVED** [review]
I'm interested in the result of testing w/o the matrix variable. Otherwise, looks great! Thanks for putting this together!

File: .github/workflows/chart-releaser.yaml (hunk 1 above)
Comment (tlangs) [reply .github/workflows/chart-releaser.yaml]: I tested it, and it seems to work without the `matrix` in the job definition. I've removed it from all the actions.

**Review by samanehsan — APPROVED** [review]
Lgtm! 🌷

**Pull request merged**